src/shared/ui/button/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './button.module.css';
+import React from "react";
+
+type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  size?: '' | 'semi' | 'large';
+  variant?: string;
```

Comment:
describe all available variants please
it should be enum
and add default value in props please

Reply:
Done